core/utils.py

```diff
@@ -6,8 +6,14 @@
 def valid_filename(s):
     return re.sub(r'(?u)[^-\w.]', '', str(s).strip().replace(' ', '_'))
 
-def generate_md(title, highlights):
+def generate_md(title, highlights, with_headings=True, normal_sorting=True):
         text = '# {}\n\n'.format(title)
+        if normal_sorting:
```

Comment:
I believe this should be a default option. I would rather remove `-s` flag from the command line arguments. It would be nice to always get the order of highlights just like in the iBooks.